.github/workflows/ci.yaml

```diff
@@ -0,0 +1,17 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: Build with Maven
+        run: mvn --batch-mode --update-snapshots package
```

Comment:
unfortunately unit tests were not run with this configuration. I guess need to add additional teps like:
`- name: Build with Maven
        run: mvn clean package
      - name: Run Unit Tests
        run: mvn test `